x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -113,6 +117,8 @@ export const ActionForm = ({
   const [actionTypesIndex, setActionTypesIndex] = useState<ActionTypeIndex | undefined>(undefined);
   const [emptyActionsIds, setEmptyActionsIds] = useState<string[]>([]);
 
+  const isSiem = featureId === 'siem';
```

Comment:
I have concerns about this approach. Having a set of overrides for SIEM specifically may be manageable on its own, but if we ever have to do this for a second feature it's going to quickly balloon out of control.

**I don't know if we need to block this PR on this, but I'm going to spell out what should be done differently. We can either refactor this PR, or open an issue to improve it on a second pass.**